app/controllers/api_controller/normalizer.rb

```diff
@@ -19,7 +19,7 @@ def normalize_hash(type, obj, opts = {})
       end
 
       attrs.each do |k|
-        value =  normalize_direct(type, k, obj[k])
+        value =  normalize_direct(type, k, obj.kind_of?(ActiveRecord::Base) ? obj.try(k) : obj[k])
```

Comment:
This make sense to me. But, I wonder if it can be simplified to always call `k` as a method. I'm not sure if we would ever get something other than an AR object, like a Hash. @abellotti what do you think?